entity-server-api/src/main/java/org/terracotta/entity/ExecutionStrategy.java

```diff
@@ -36,6 +36,8 @@ public boolean runOnActive() {
       }
     }, BOTH {
 
+    }, ALL {
```

Comment:
javadoc says: `on the active server, passive server or both`. So what `ALL` means ?

Reply:
ALL and BOTH are synonyms